Add payment method for wechat & fixing a minor typo

abigail830 opened the pull request:
The CI would pass once the bisq-core change get merged to master and rebuild. Now just pending that dependency ready
![testing](https://user-images.githubusercontent.com/19251334/38478543-015d0c38-3bec-11e8-955a-cb6dfca97e2a.jpg)

abigail830 committed:
Fixing a minor typo for validator param

abigail830 committed:
Adding payment method: Wechat

ripcurlx committed:
Changed import style

ripcurlx commented on src/main/java/bisq/desktop/main/account/content/fiataccounts/FiatAccountsView.java:
```diff
@@ -17,93 +17,39 @@
-import bisq.desktop.components.paymentmethods.UpholdForm;
-import bisq.desktop.components.paymentmethods.VenmoForm;
-import bisq.desktop.components.paymentmethods.WesternUnionForm;
+import bisq.desktop.components.paymentmethods.*;
```
Could you please use imports like @cbeams mentioned in https://github.com/bisq-network/bisq-core/pull/70#pullrequestreview-110353024. I've fixed the imports in my local repo already. Could you please check "[Allow edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)" so I can push my local changes to your fork?

abigail830 replied:
Already kicked 'Allow edits from maintenaners' please check if it is ok.

Just curious, are you doing this by manually update import and not select the 'optimize import' during commit?

Because I had tried again in my workspace, no matter using auto import or optimize import during commit, it would update into .* style..

ripcurlx replied:
Worked for me thanks! I'm not committing directly from Intellij, but from and external tool (just my personal habit), so I guess that's why the `optimize import` never occurred to me. But good to know if I change my worfklow.

As soon as https://github.com/bisq-network/bisq-core/pull/70#pullrequestreview-110520877 is changed this PR is ACK from my side. Did one trade on RegTest with the changes.

ripcurlx replied:
For your next contributions, please use signed commits as explained here: https://help.github.com/articles/signing-commits-using-gpg/. Then you'll have the nice `Verified` label next to your commits 😄

abigail830 committed:
Optimize imports

abigail830 committed:
Optimize imports

abigail830 commented:
Updated import style as the same. And also add u access right in my fork. Please kindly help to review @ripcurlx

cbeams committed:
Merge branch 'master' into abigail830-master

cbeams committed:
Polish WeChat payment method support

 - Use camel case spelling of 'WeChat' consistently
 - Organize imports with common layout (bisq-network/style#2)

cbeams reviewed (approved):
utACK. See my review changes in 95daeea. @abigail830, in the future, please:

 - submit pull requests from a topic branch in your fork (instead of `master`)
 - issue separate changes as separate pull requests. It's great that you caught the typos in `CashAppForm` and `MoneyBeamForm`, etc, but would be better if those fixes had been submitted in their own PR, separate from this one.

@ripcurlx, please merge whenever you see fit.

abigail830 commented:
@cbeams got it and totally agree!
